src/integrations/GA4/browser.js

```diff
@@ -19,10 +19,10 @@ export default class GA4 {
       logger.setLogLevel(analytics.logLevel);
     }
     this.name = NAME;
-    this.sessionId = '';
     this.analytics = analytics;
     this.measurementId = config.measurementId;
     this.capturePageView = config.capturePageView || 'rs';
+    this.overrideSessionId = config.overrideSessionId || false;
     this.extendPageViewParams = config.extendPageViewParams || false;
     this.isHybridModeEnabled = config.useNativeSDKToSend === false || false;
```

Comment:
can we start using `connectionMode` to check isHybridModeEnabled ?

Reply:
we are not receiving `connectionMode` in config, probably need to check with @sandeepdsvs